### ISSUE TITLE
zm_wb_daemon restart leaves unfinished jobs

### ISSUE THREAD

**Issue by aabdnn** [dotse/zonemaster-backend#272]
I've just noticed something. If I have tests running (submitted in a big batch, for example), and I restart zm_wb_daemon while they're being processed, then the jobs that were incomplete never get restarted.

I have to perform manual surgery on the database with:

```
update test_results set progress=0 where progress>0 and progress<100
```

Is it not be better for the zm_wb_daemon workers to reset "progress" to 0 themselves when they die before the tests are complete? Then, the tests can just start running again themselves.

**Comment by matsduf** [comment]
We need a solution that will handle all situation, i.e. also when the daemon dies brutally,

**Comment by aabdnn** [comment]
That's true. The current model does not allow clean continuation from abrupt shutdowns or crashes. I don't have an immediate solution in mind yet.
